.circleci/config.yml

```diff
@@ -2,28 +2,28 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/node:8.9.1-browsers
+      - image: leeaandrob/paratii-portal:0.0.5
+        env:
+          - DISPLAY=:99
+          - CHROME_BIN=/usr/bin/google-chrome
     working_directory: ~/repo
     steps:
       - checkout
-      - restore_cache:
-          keys:
-          - v1-dependencies-{{ checksum "package-lock.json" }}
-          # - v1-dependencies-
+      # - restore_cache:
+      #     keys:
+      #     - v1-dependencies-{{ checksum "package-lock.json" }}
+      #     - v1-dependencies-
```

Comment:
Should we be ignoring the dependencies cache for now? If so, I would just delete these rather than leave comments in the source